spec/chewy/type/import_spec.rb

```diff
@@ -43,6 +43,45 @@
       end
     end
 
+    describe 'using scopes with includes' do
+      if const_defined?('::ActiveRecord') && ::ActiveRecord::VERSION::MAJOR > 4
```

Comment:
Using something like this

```ruby
describe 'using scopes with includes', :active_record do
  if ::ActiveRecord::VERSION::MAJOR > 4
  ...
  end
end
```

does not work for the non-AR gemfiles and something like

```ruby
specify do
  skip unless ::ActiveRecord::VERSION::MAJOR > 4
  ...
end
```

does not work for the ar-3 gemfiles, because the `before` block is evaluated before one is able to skip-out and it will raise, because it cannot handle `where.not` correctly, which requires AR >= 4.